tests/echo_service_tests.rs

```diff
@@ -0,0 +1,40 @@
+use rusk::services::echoer::basic_proto::echoer_client::EchoerClient;
+use rusk::services::echoer::basic_proto::echoer_server::EchoerServer;
+use rusk::services::echoer::basic_proto::EchoRequest;
+use rusk::services::echoer::Rusk;
```

Comment:
`Rusk` shouldn't be part of `echoer`: ideally `Rusk would implements several services. So it should be probably at the root level:
```rust
`use rusk::Rusk;`
```

Reply:
Done.